_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,251 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Before you can create [notes](/_docs/concepts/what-is-grafeas/overview.md) or
+[occurrences](/_docs/concepts/what-is-grafeas/overview.md), you'll need a
+running Grafeas server, which you can set up following
+[these instructions](https://github.com/grafeas/grafeas/blob/master/docs/running_grafeas.md).
+The examples below assume that you have a local server running on port 8080.
+
+## Create a project
+
+All notes and occurrences must be stored in a project. Generally, you should
```

Comment:
Create separate projects for notes and occurrences so that setting up fine-grained access control is easier.